aries-site/src/examples/templates/forms/do-dont/ColumnFormDont.js

```diff
@@ -0,0 +1,299 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  MaskedInput,
+  ResponsiveContext,
+  Select,
+  Text,
+  TextInput,
+} from 'grommet';
+import { emailMask, emailValidation, phoneMask } from '../utils/FormValidation';
+
+const states = [
+  'AK',
+  'AL',
+  'AR',
+  'AZ',
+  'CA',
+  'CO',
+  'CT',
+  'DC',
+  'DE',
+  'FL',
+  'GA',
+  'HI',
+  'IA',
+  'ID',
+  'IL',
+  'IN',
+  'KS',
+  'KY',
+  'LA',
+  'MA',
+  'ME',
+  'MD',
+  'MN',
+  'MO',
+  'MS',
+  'MT',
+  'NC',
+  'ND',
+  'NE',
+  'NH',
+  'NJ',
+  'NM',
+  'NY',
+  'NV',
+  'OH',
+  'OK',
+  'OR',
+  'PA',
+  'RI',
+  'SC',
+  'SD',
+  'TN',
+  'TX',
+  'UT',
+  'VA',
+  'VT',
+  'WA',
+  'WI',
+  'WV',
+  'WY',
+];
+
+export const ColumnFormDont = () => {
+  const [formValues, setFormValues] = React.useState({
+    firstName: '',
+    lastName: '',
+    address1: '',
+    address2: '',
+    city: '',
+    state: '',
+    zipcode: '',
+    isBusiness: '',
+    'fullName-shipping': '',
+    'phone-shipping': '',
+    'email-shipping': '',
+  });
+  const size = useContext(ResponsiveContext);
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  // provide order of formfields for validation
+  // to properly place focus on any errors or infos
+  const formFields = [
+    'firstName',
+    'lastName',
+    'address1',
+    'address2',
+    'city',
+    'state',
+    'zipcode',
+    'isBusiness',
+    'fullName-shipping',
+    'phone-shipping',
+    'email-shipping',
+  ];
+
+  // On long forms, we want to focus the first of any fields that
+  // return an error or info message. This removes the need for the
+  // user to scroll to find which field blocked form submission.
+  const onValidate = validationResults => {
+    const target = formFields.find(
+      field =>
+        field in validationResults.errors || field in validationResults.infos,
+    );
+    if (target) {
+      const targetFormField = document.getElementsByName(target);
+      targetFormField[0].focus();
+    }
+  };
+
+  return (
```

Comment:
so I was going back and forth with this since the shipping example is what we want to show for users as far as code goes so I didn't want to make the original example be overcomplicated but we can export shipping info and contact info that may be easier that way I can have the extra `boxes` in this file and no one will know